modules/structs/issue_label.go

```diff
@@ -47,8 +47,9 @@ type EditLabelOption struct {
 
 // IssueLabelsOption a collection of labels
 type IssueLabelsOption struct {
-	// list of label IDs
-	Labels []int64 `json:"labels"`
+	// Labels can be a list of intergers representing label IDs
```

Comment:
Small typo nitpick:
```suggestion
	// Labels can be a list of integers representing label IDs
```

Looks like you would also need to regenerate this in the swagger .tmpl file